internal/contracts/manager.go

```diff
@@ -299,6 +308,17 @@ func (cm *contractManager) resolveInvokeContractRequest(ctx context.Context, req
 		if err != nil || req.Method == nil {
 			return i18n.NewError(ctx, coremsgs.MsgContractMethodResolveError, err)
 		}
+		errors, err := cm.database.GetFFIErrors(ctx, cm.namespace, req.Interface)
+		if err != nil {
+			return i18n.NewError(ctx, coremsgs.MsgContractErrorsResolveError, err)
+		}
+		if req.Options == nil {
+			req.Options = map[string]interface{}{
+				"errors": errors,
+			}
+		} else {
+			req.Options["errors"] = errors
+		}
```

Comment:
Perhaps instead of setting `req.Options["errors"]` here, we should add an `errors` field to the `ContractCallRequest` struct itself.